Library/Formula/subversion.rb

```diff
@@ -70,6 +70,18 @@ class Subversion < Formula
   end
 
   def install
+    # OS X's Python is built universally and can't link with Homebrew's deps
+    # unless Homebrew's deps are universal as well.
+    # https://github.com/Homebrew/homebrew-versions/issues/777
+    if build.with?("python") && !File.exist?(HOMEBREW_PREFIX/"bin/python")
+      unless build.universal?
```

Comment:
I presumed OS X's Python would always be universal, and we weren't keen on supporting the Python we warn about in `brew doctor` so I worded it like this to try and "enforce" a "friendly" Python. But I'm cool with the change - I'd pretty much jump through a glass window before going against one of your Python-handling recommendations :smile_cat: